src/pages/Home/Home.page.jsx

```diff
@@ -1,26 +1,44 @@
-import React from 'react';
-import styled from 'styled-components';
+import React, { useState, useEffect } from 'react';
 import Header from '../../components/Header';
 import Content from '../../components/Content';
+import { Homepage, InputSearch } from './Home.styled';
 
-const Homepage = styled.section`
-  width: 100%;
-  text-align: center;
-  justify-content: center;
-  &:h2 {
-    font-size: 3.75rem;
-    font-weight: 300px;
-    line-height: 1.2;
-    font-family: 'Roboto', 'Helvetica';
-  }
-`;
+import useYoutubeApi from '../../components/helpers/useYoutubeApi';
 
 function HomePage() {
+  const { data, loading, error, fetchSearch } = useYoutubeApi();
+  const [value, setValue] = useState('wizeline');
+
+  useEffect(() => {
+    if (data.length === 0) {
+      fetchSearch(value);
+    }
+    if (error !== null) {
+      console.log(error);
+    }
+  }, [value, data, fetchSearch, error]);
+
+  const enterPressed = async (event) => {
```

Comment:
you have two different `enterPressed` that I think could be only one function living inside the `Header` component, so instead of passing `enterPressed` you can pass `value` and move `enterPressed` to the `Header` component